pyabc/smc.py

```diff
@@ -225,6 +225,14 @@ def __init__(self,
         self.min_acceptance_rate = None
 
         self._sanity_check()
+        self.aknowledge_n_procs()
+
+    def aknowledge_n_procs(self):
```

Comment:
maybe this can be done at the end of the pyabc.sampler.multicorebase ``__init__`` function? Would make sense if the check happens in the constructor method of the sampler.

Reply:
What do you mean? now the call of the function happened on the __init__ function.

Reply:
I mean not have the function in here, but in pyabc.sampler.multicorebase (and there call it in the class's ``__init__`` function), so that that funcionality is encapsulated.